solr/solr-ref-guide/modules/deployment-guide/pages/enabling-ssl.adoc

```diff
@@ -245,7 +245,7 @@ Windows::
 ====
 [source,powershell]
 ----
-C:\> bin/solr.cmd --property urlSchema --value https --zk-host server1:2181,server2:2181,server3:2181
```

Comment:
Does this need a `bin/solr cluster` here?

Reply:
I did not notice that. As I don't have a windows system I can't really confirm this. The script is running the cluster command with the solrcli class. So probably would be documented there.

https://github.com/apache/solr/blob/main/solr/core/src/java/org/apache/solr/cli/SolrCLI.java

and then uses:

https://github.com/apache/solr/blob/main/solr/core/src/java/org/apache/solr/cli/ClusterTool.java

SolrCli has a cluster command so probably this needs cluster too, but I have found no documentation of SolrCLI at all.